__e2e__/peruse.spec.js

```diff
@@ -242,8 +244,84 @@ describe( 'main window', () =>
         expect( bookmarks ).toMatch( 'shouldappearinbookmarks' );
 
     } );
+    it( 'can check if settings menu exists', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
+
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        const menuExists = await client.waitForExist( BROWSER_UI.SETTINGS_MENU__BUTTON, WAIT_FOR_EXIST_TIMEOUT );
+        expect( menuExists ).toBeTruthy();
+    } );
 
+    it( 'can open settings menu', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
 
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        await client.click( BROWSER_UI.SETTINGS_MENU__BUTTON );
+        await delay( 4500 );
+        const settings = await client.waitForExist( BROWSER_UI.SETTINGS_MENU, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
+        expect( settings ).toBeTruthy();
+    } );
+    it('can open settings menu and checks if Bookmarks,History,Toggle exist', async () => 
+    {
+        expect.assertions( 4 );
+        const { client } = app;
+        await delay( 4500 );
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        await client.click( BROWSER_UI.SETTINGS_MENU__BUTTON );
+        await delay( 4500 );
+        const settings = await client.waitForExist( BROWSER_UI.SETTINGS_MENU, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
+        const bookmarks = await client.waitForExist( BROWSER_UI.SETTINGS_MENU__BOOKMARKS, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
+        const history = await client.waitForExist( BROWSER_UI.SETTINGS_MENU__HISTORY, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
+        const toggle = await client.waitForExist( BROWSER_UI.SETTINGS_MENU__TOGGLE, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
+        expect( settings ).toBeTruthy();
+        expect( bookmarks ).toBeTruthy();
+        expect( history ).toBeTruthy();
+        expect( toggle ).toBeTruthy();
+    } );
+    it('can open settings menu and navigate to bookmarks', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        await client.click( BROWSER_UI.SETTINGS_MENU__BUTTON );
+        await delay( 4500 );
+        await client.click( BROWSER_UI.SETTINGS_MENU__BOOKMARKS );
+        await delay( 4500 );
+        const header = await client.getText( 'h1' );
+        await delay( 2500 );
+        expect( header ).toBe( 'Bookmarks' );
+    } );
+    it('can open settings menu and navigate to history', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
```

Comment:
unless the CI is throwing errors, let's leave a smaller delay in general please, as `4500` is quite long, especially cumulatively.

Reply:
Noted I've used the delays in other tests as reference for these tests. But in general I agree that it should be as minimal as possible.